travis/vars.yaml

```diff
@@ -2,6 +2,7 @@ extends: vars_testgeomapfish.yaml
 
 vars:
     host: localhost
+    db: geomapfish_test
```

Comment:
If I remove this then I have an error when calling `travis/create-new-project.sh`  in Travis:
```
sqlalchemy.exc.OperationalError: (psycopg2.OperationalError) FATAL:  database "geomapfish" does not exist
```
(https://travis-ci.org/camptocamp/c2cgeoportal/builds/189114917)

Is there a better way to avoid this?